client/src/constants/backend_constants.js

```diff
@@ -0,0 +1,759 @@
+import deepFreeze from 'deep-freeze';
+
+export default deepFreeze({
```

Comment:
@Tengelma is this something we can do now?

Reply:
I think so? There are a lot of items in BACKEND_CONSTANTS that weren't automatically generated but if we a) move as much of that stuff as we can out and b) modify the script we used to generate this initially to get everything into an usable format, it's definitely possible. I think Cosmology is the main thing that didn't make it with the original script so that will need some work. We might even be getting to a point where we could support more generic form templates like you suggested early on @aneuhold.

I think it would be a 'one and done' sort of thing where that script runs when starting the server and saves the constants somewhere so we can grab the same object with some initial_data() endpoint. Can tinker with this next sprint?

Reply:
@Tengelma absolutely. Sounds like a great idea generating on server start, because of course the library isn't going to change in memory while the server is running. haha. Didn't think of that at first so good pointing that out. Also looking forward to discussing this tomorrow 😄

Reply:
That sounds like the right plan. Things like the `backend_constants` and the initial plot data should be generated on server start.